### PR TITLE
Add argument to only evaluate single-speaker regions

### PR DESCRIPTION
**Pull request by thequilo**
Adresses #62. The name is still up for discussion.

### CODE REVIEW

File: meeteval/der/__main__.py [hunk 1]
```diff
@@ -24,7 +26,20 @@ def md_eval_22(
...
     from meeteval.wer.__main__ import CLI
 
-    cli = CLI()
+    class DerCLI(CLI):
+        def add_argument(self, command_parser, name, p):
+            if name == 'exclude_overlap':
+                command_parser.add_argument(
+                    '-1', '--exclude-overlap',
+                    action='store_true',
+                    help='Limits scoring to single-speaker regions. '
```
Comment (boeddeker) [inline meeteval/der/__main__.py:35]: @TCord Could you add a sentence, where the user should be careful? Or was that only relevant, if you want to calculate the overlap only DER?

**Comment by boeddeker** [comment]
I vote for`--regions [all|single|overlap|nonoverlap]` instead of adding a new option that is not used in any existing framework.
It is already used in spyder and works for md-eval, dscore, pyannote and spyder.

**Comment by thequilo** [comment]
I also like `--regions`, but md-eval only supports `--regions [all|nooverlap]`. Also, should we name it `--score-regions`?

**Comment by boeddeker** [comment]
> I also like --regions, .... Also, should we name it --score-regions?

Hmm. My motivation was to use the same name as others.

> but md-eval only supports `--regions [all|nooverlap]`

That shouldn't be a problem.

**Comment by thequilo** [comment]
> Hmm. My motivation was to use the same name as others.

`--regions` is not as self-explanatory, `--score-regions` gives a much better hint at what this "region" means. But since all names we found so far have problems, I'm pretty sure that anything we'll come up with will also have its flaws, so I'll stick with `--regions`.

**Review by boeddeker — APPROVED** [review]

**Pull request merged**